javalin/src/main/java/io/javalin/http/JavalinServletHandler.kt

```diff
@@ -23,9 +23,14 @@ data class Stage(
 
 internal data class Result<VALUE : Any?>(
     val previous: InputStream? = null,
-    val future: CompletableFuture<VALUE> = completedFuture(null),
+    val future: CompletableFuture<VALUE>? = null,
     val callback: Consumer<VALUE>? = null,
-)
+) {
+
+    fun futureOrCompletedStage(): CompletableFuture<VALUE> =
+        future ?: completedFuture(null)
+
+}
```

Comment:
Is this function necessary? Does it make more sense than simply calling 

```
(result.future ?: completedFuture(null)).thenAccept
```

further down?